deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -1132,6 +1102,27 @@ def find_cve_severity_score (cve_array, cve_id):
 
     return severity, cvss2_score, cvss3_score
 
+def find_rhel_cve_severity_score(cve_nvd_array, cve_rhel_array, cve_id):
+    """It looks for the cve_id in cve_array and returns its severity and score
+
+    Args:
+        cve_array (str) = The CVE's data read from the RHEL feed.
+        cve_id (str) = The CVE ID to find and return its severity and score.
+    """
+    severity, cvss2_score, cvss3_score = find_cve_severity_score(cve_nvd_array, cve_id)
+
+    for cve in cve_rhel_array:
+        if cve['CVE'] == cve_id:
+            # The CVSS3 has priority over CVSS2
+            if cve['cvss_score'] is not None:
```

Comment:
I think it is not the same. With `cve['cvss_score'] is not None` we're evaluating whether the value of the key `cvss_score` exists or not, but with `cvss_score in cve` we're evaluating if the dict `cve` contains a key called `cvss_score`. 
If we have `{"cvss_score":None}` using the first conditional will ignore the if, with the suggested changed it will execute the if statement and it will assign a cvss_score2 = None.

Reply:
You are right! These are different things.
Maybe, we can check both.

Reply:
Yes, after the discussion we had I added that check too. Thanks!